### PR TITLE
Add Driver Control Commands (v1)

### PR DESCRIPTION
**Pull request by duane701**
Add commands to support drivers controls.

Replaces #12 .

### CODE REVIEW

File: src/OI.cpp [hunk 1]
```diff
@@ -8,59 +8,95 @@
...
+  // SmartDashboard Buttons
+  SmartDashboard::PutData("TankDrive", new TankDrive());
+  SmartDashboard::PutData("Autonomous Command", new AutonomousCommand());
+  SmartDashboard::PutData("ToggleShifter", new ToggleShifter());
+  SmartDashboard::PutData("ToggleLifter", new ToggleLifter());
+  SmartDashboard::PutData("ToggleArms", new ToggleArms());
 }
 
 std::shared_ptr<Joystick> OI::getDriver() {
    return driver;
```
Comment (Diego-Andrade) [inline src/OI.cpp]: Three spaces?

**Review by Diego-Andrade — COMMENTED** [review]
Besides what looks like a leftover space from autogenerated code, LGTM

**Comment by duane701** [comment]
PTAL

If this is good, you have to "approve" the PR.

**Comment by Diego-Andrade** [comment]
LGTM. Does that mean I can now merge and close the PR?

**Comment by duane701** [comment]
Yes it does. But you need to approve this PR first.

**Review by Diego-Andrade — APPROVED** [review]
Approving PR

**Pull request merged**